README.md

```diff
@@ -2,6 +2,161 @@
 
 # Task estimation
 
+### Project estimation
+**What's a project ?**
+A project is defined by "an individual or collaborative enterprise that is carefully planned to achieve a particular aim".
+
+Before defining a task, we need to define the overall project. That is, we need to need to think about several points that will help us to better define later the project tasks and their estimation.
```

Comment:
Little human error here, "we need to need" should be "we need to"

Reply:
Indeed ! I'll fix it